tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "module":"CommonJS",
+    "target": "es2019",
```

Comment:
Does it mean the published npm package supports es2019 only? I thought it'll be more compatible targeting es6. Basically I think it should support the lowest Node.js version in maintenance state ([which is 10 currently](https://nodejs.org/en/about/releases/))

Reply:
Ok, I will change it :]

WDYT regarding the code it self?
I'm not sure what to test, do you have any pointers? (I'm noob in Shopify themes).

Reply:
Looks good to me.

Reply:
Based on  https://stackoverflow.com/questions/51716406/typescript-tsconfig-settings-for-node-js-10
 It should be es2018